### PR TITLE
{geo} [intel/2016b] SimPEG/0.3.1 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4a1bd66a357f85ecbe8027d4cc6d8b7f for a full test report.

File: easybuild/easyconfigs/s/SimPEG/SimPEG-0.3.0-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://simpeg.xyz'
+description = """Simulation and Parameter Estimation in Geophysics: a python package for simulation and gradient
+based parameter estimation in the context of geophysical applications."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/simpeg/simpeg/archive/']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SimPEG/SimPEG-0.3.0-intel-2016b-Python-2.7.12.eb]: PyPI has ` 0.3.1` release?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2007.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3e419a5657a7d4584ade417376f9cfc5 for a full test report.

File: easybuild/easyconfigs/s/SimPEG/SimPEG-0.3.0-intel-2016b-Python-2.7.12.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/s/SimPEG/SimPEG-0.3.0-intel-2016b-Python-2.7.12.eb]: not on github...

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c13c38917de7f2cb48e122c5627493c1 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cd9411fbcf00c1499f5415f5da7aa647 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**